apps/parser/.gitignore

```diff
@@ -0,0 +1,49 @@
+node_modules
+.vscode
+.DS_Store
+/docs/api.md
+/coverage
+/lib
+/esm
+/cjs
+/browser
+*.tgz
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# Dependencies
+node_modules
+.pnp
+.pnp.js
+
+# Local env files
+.env
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+# Testing
+coverage
+
+# Turbo
+.turbo
+
+# Vercel
+.vercel
```

Comment:
Why adding so much things to the .gitignore? What are those additions based on?

For example, the `.next` dir?

Reply:
removed unwanted keys.